Gruntfile.js

```diff
@@ -184,6 +199,33 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-open');
   grunt.loadNpmTasks('grunt-browserify');
 
-  grunt.registerTask('default', ['exec:bower_update','clean','copy:fonts','sass:dev','browserify:dev','usebanner','exec:build_sphinx','connect','open','watch']);
-  grunt.registerTask('build', ['exec:bower_update','clean','copy:fonts','sass:build','browserify:build','uglify','usebanner','exec:build_sphinx']);
+  grunt.registerTask('default', [
+      'exec:bower_update',
+      'clean',
+      'copy:fonts',
+      'sass:dev',
+      'browserify:dev',
+      'usebanner',
+      'exec:build_sphinx',
+      'connect',
+      'open',
+      'watch'
+  ]);
+  grunt.registerTask('build', [
+      'exec:bower_update',
+      'clean',
+      'copy:fonts',
+      'sass:build',
+      'browserify:build',
+      'uglify',
+      'usebanner',
+      'exec:build_sphinx'
+  ]);
+  grunt.registerTask('i18n', [
+      'exec:babel_extract',
+      'exec:tx_push',
+      'exec:tx_pull',
+      'exec:babel_compile',
+      'exec:build_sphinx:es'
```

Comment:
This hardcodes the translated language to use and will require a manual update on every new language. Is there a way to get all available languages automatically so the Gruntfile is always in sync with the translations?

Reply:
Oh, i should have explained. `build_sphinx:es` is just to test out that translations are working, it isn't actually used anywhere. With the webpack changes, this will change again -- do we want to keep the test build step?

Reply:
Because grunt is going away, and this is a packaging step of the theme, I've moved these to a setup.py command: `update_translations`. I dropped building the sphinx docs as a test